element_moseq/moseq_train.py

```diff
@@ -426,12 +444,14 @@ class PreFitTask(dj.Manual):
     """
 
     definition = """
-    -> PCAFit                                       # `PCAFit` Key
-    pre_latent_dim               : int              # Latent dimension to use for the model pre-fitting
-    pre_kappa                    : float            # Kappa value to use for the model pre-fitting
-    pre_num_iterations           : int              # Number of Gibbs sampling iterations to run in the model pre-fitting
+    -> PCAFit                                           # `PCAFit` Key
+    pre_latent_dim               : int                  # Latent dimension to use for the model pre-fitting
+    pre_kappa                    : float                # Kappa value to use for the model pre-fitting

```

Comment:
Typically, for these types of cases, we define a `ParamSet` table that combines multiple parameters into a single entry with an short primary key.

Reply:
That's a great suggestion. Instead of using floats for the primary key, we can define the `pre_kappa` and `full_kappa` values as integers since they are already being converted to floats in the specific functions within the `make` function. This change will make equality comparisons easier. Additionally, by making this adjustment, we can avoid the need to create a separate `ParamSet` table in the pipeline architecture. These changes have been addressed in [this commit](https://github.com/datajoint/element-moseq/pull/4/commits/fdb08fe307215c8094d189e9038aa4a22b86923b).